app/pages/help.html

```diff
@@ -45,6 +45,9 @@ <h1 id="intro">Intro</h1>
             <li><kbd>e</kbd> - Switch to <a class="explore" href="#action.toExploreMode">explore mode</a> (for navigating to websites or searching the web)</li>
             <li><kbd>f</kbd> - Switch to <a class="follow" href="#action.startFollowCurrentTab">follow mode</a> (for clicking buttons or links based on keys)</li>
             <li><kbd>/</kbd> - Switch to <a class="search" href="#action.toSearchMode">search mode</a> (for locating text in the page, use <kbd>n</kbd> and <kbd>N</kbd> to switch between matches)</li>
+            <li><kbd>v</kbd> - Switch to <a class="pointer" href="#pointer.start">pointer mode</a> (for hovering over links, downloading images, copying link urls, etc.)<ul>
```

Comment:
Can we link this to `#pointer` instead of the start action specifically?

Reply:
Sure, I just tried to be consistent with the other items.